flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/CalcITCase.scala

```diff
@@ -66,7 +66,7 @@ class CalcITCase extends StreamingTestBase {
     result.addSink(sink)
     env.execute()
 
-    val expected = List("I(1,1,1)")
+    val expected = List("I+(1,1,1)")
```

Comment:
after looking at `I+(1,Alice)`  I’m a bit concerned because it looks like math
other way around `+I(1,Alice)` ?

Reply:
```
+I | a | 12 | FDF
-U | b | 13 | EFC
+U | b | 14 | CDE
-D | d | 15 | IKH
```